components/Layout/Banner/LayoutBannerNewLearningPlatform.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <qiskit-banner>
+    Introducing&nbsp;
+    <a href="#">IBM Quantum Learning</a>, a new home for free quantum education
+    resources. Continue to use Learn on Qiskit until December 1st, 2023
+  </qiskit-banner>
+</template>
+
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+qiskit-banner {
+  display: inline-block;
+  width: 100%;
+  background: carbon.$cool-gray-20 0%;
```

Comment:
any reason to not use `background-color: carbon.$cool-gray-20;`?

Reply:
good catch! Updated here https://github.com/Qiskit/qiskit.org/pull/3411/commits/f881264b2497b8eed4d35f56368eadcdb8e07060